continuous-integration/reference-architectures/README.md

```diff
@@ -1,3 +1,181 @@
 # Reference Architectures
 
-Nothing here yet, come back soon... 
\ No newline at end of file
+## Overview
+Architectures presented here provides a high-level reference guide for the implementation of 
+continuous integration best practices. They describe the structure, component interrelationship and 
+processes invoked without detailing any specific implementation. Please see categories and links 
+below for details.
+
+## Continuous Integration
+Continuous integration (CI) is an automated process triggered by development changes to perform an 
+end-to-end validation and presentation of software. There are a number of different steps whose goal 
+is to ensure code works together, tests successfully and releases readily. CI is implemented as a 
+_development pipeline_ that automates steps normally undertaken in a release process.
+
+### An approach to best practices
```

Comment:
"An approach to best practices" is kind of confusing. What does that mean? Suggestion to reword the title to something like: "Background"

Reply:
Done. Will be in next commit.